lib/engine/game/g_1846.rb

```diff
@@ -330,7 +336,24 @@ def event_remove_tokens!
       end
 
       def bankruptcy_limit_reached?
-        @bankruptcies >= @players.size - 1
+        @bankrupt_players.size >= (all_players.size - 1)
+      end
+
+      def sellable_bundles(player, corporation)
+        return [] if corporation.owner == @share_pool
+
+        super(player, corporation)
+      end
+
+      def bundle_is_presidents_share_alone_in_pool?(bundle)
```

Comment:
name is ridiculous lol

Reply:
but you know what it does!

Reply:
happy for suggestions for a concise name